src/share/classes/com/sun/javatest/regtest/tool/JarManager.java

```diff
@@ -73,13 +73,12 @@ static Path forClass(Class<?> clazz) {
     JarManager(Path libDir) {
 
         InputStream in = getClass().getResourceAsStream("jars.properties");
-        if (in == null) {
-            throw new Error("Can't find jars.properties");
```

Comment:
What's the reason for dropping this?

Reply:
I guess it is expected to be available, by coo situation in the build, so this was effectively just an assertion, right?

Reply:
Yes. Without this assertion in place, an IDE-based launch of `jtreg`'s main program is possible. For example, here are the options for a test run for all self-tests in `test/junit`:
```
-Djunit.path=<ABSOLUTE-PATH-TO>/jtreg/build/deps/junit/junit-platform-console-standalone-1.10.2.jar
com.sun.javatest.regtest.Main
-v:pass,error,fail
test/junit
```
I tried to provide an empty `jars.properties` instead of removing the check, but that didn't work out.

Reply:
Which tests are you hoping to run by having the IDE invoke the main program directly?  And how about you hoping to invoke `jtreg` -- on the source directory or on the generated `.jar` files?

If you are hoping to run JDK tests, I can believe that the IDE can make JUnit and TestNG available -- but what about asmtools -- which is also normally in `jars.properties`.

Reply:
Without addressing the need for `asmtools`, you're at risk for an NPE in `ActionRecorder.java:157`

Reply:
And it is unrelated to this verbose flag issue anyway. ;-) So, I'll remove those changes from this PR.